resources/packs/python/python.lr

```diff
@@ -0,0 +1,22 @@
+
+import "../core/core.lr"
+
+// Python packages resource
```

Comment:
nit: this resource describes the current operating system's Python installation, its packages and path etc

Reply:
Yes, I went and added all the comments for the auto-docs while I was there as well.